### ISSUE TITLE
No FWHM infos for Litebird instrument

### ISSUE THREAD

**Issue by giuspugl** [dpole/cmbdb#3]
I recently noticed  that FWHM attribute for Litebird is missing . Is there any reason why ?

**Comment by dpole** [comment]
Congrats! First issue of the repo!

No reason. I only updated LiteBIRD with minimal information from the paper submitted to PTEP. 

I didn't need the beam info. I procrastinated adding that info because I prefer working with channels at the same frequency combined. However, the channels have different beams. Any opinion about which choice would be most useful?

**Comment by giuspugl** [comment]
> the channels have different beams

yeah, good point !
> Any opinion about which choice would be most useful?

since the different reso in the same freq. channel  are not very dramatic, how about considering just the largest beam ?

**Comment by dpole** [comment]
Yeah, let's go that way. Please, pull and try. Let me know how it goes.

**Comment by giuspugl** [comment]
everything looks good!
